java/runtime/src/main/java/org/ray/runtime/raylet/MockRayletClient.java

```diff
@@ -20,11 +23,13 @@
   private final Map<UniqueId, Map<UniqueId, TaskSpec>> waitTasks = new ConcurrentHashMap<>();
```

Comment:
1) rename this to `waitingTasks`
2) Why is this a `Map<UniqueId, Map<UniqueId, TaskSpec>>`, instead of just `Map<UniqueId, TaskSpec>`?

Reply:
yes, I think Map<UniqueId, TaskSpec> is more suitable too